src/components/City/City.js

```diff
@@ -2,11 +2,37 @@ import React from "react";
 import './City.css';
 
 import Weather from "../Weather/Weather";
+import Header from "../Header/Header";
+
+import sunBg from '../../assets/weather-bgs/sunny-bg.jpeg';
+import cloudBg from '../../assets/weather-bgs/cloudy-bg.jpeg';
+import rainBg from '../../assets/weather-bgs/rainy-bg.jpg';
+import snowBg from '../../assets/weather-bgs/snowy-bg.jpeg';
+import stormBg from '../../assets/weather-bgs/thunderstorm-bg.jpeg';
+import fogBg from '../../assets/weather-bgs/foggy-bg.jpeg';
+
+const City = ({ weather, type, city }) => {
+
+  const pickBg = () => {
+    if (type.includes('Sunny') || type.includes('Clear')) {
+      return sunBg;
+    } else if (type.includes('Cloudy')) {
+      return cloudBg;
+    } else if (type.includes('Snow') || type.includes('Ice') || type.includes('Flurries')) {
+      return snowBg;
+    } else if (type.includes('Rain') || type.includes('Drizzle')) {
+      return rainBg;
+    } else if (type.includes('Thunderstorm')) {
+      return stormBg;
+    } else if (type.includes('Fog')) {
+      return fogBg;
```

Comment:
I remember you mentioning not being super happy with this piece of code...

I haven't dug into this too much for a proof of concept, so I may be missing a crucial piece, but how about if you built out an object with your types and their associated backgrounds, then referenced that property by it's type key to get back the background you needed. eg:


`const bkgdObk = {`
`  Cloudy: cloudbkgd,`
`  Sunny: sunbkgd,`
`etc....`
`}`

`url(${bkgdObj[type]})`


Let me know if that makes sense and/or if you wanna chat through it!